Gopkg.lock

```diff
@@ -2,16 +2,27 @@
 
 
 [[projects]]
-  digest = "1:4d2e5a73dc1500038e504a8d78b986630e3626dc027bc030ba5c75da257cdb96"
+  digest = "1:ffe9824d294da03b391f44e1ae8281281b4afc1bdaa9588c9097785e3af10cec"
+  name = "github.com/davecgh/go-spew"
```

Comment:
Looks like this was used for debugging during development only (I don't see it referenced anywhere); please remove.

Reply:
Yes, my bad - removing.

Reply:
Done